### PR TITLE
185 step3 s3 bucket signed url, dynamodb creation, sqs ,sns, discord webhook integration

### PR DESCRIPTION
**Pull request by faisalisafk**
Fixes #185 

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first:

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
Added s3 bucket,dynamodb table, SQL, and SNS event policy with s3 backend integration
## What was done
- Added s3 bucket with s3 delete event trigger for lambda
- Added new dynamodb table and set policy accordingly
- Created SQS event and attached with stream event 
- Created SNS event and published message on the discord webhook
- Added S3 Backend with my S3 bucket
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

**Review by mhazad — APPROVED** [review]
LGTM

**Review by mhazad — APPROVED** [review]
LGTM

File: faisal/.gitignore [hunk 1]
```diff
@@ -20,7 +20,17 @@ var/
...
\ No newline at end of file
+terraform/.terraform.lock.hcl
+=======
+# Local .terraform directories
+**/.terraform/*
+.terraform.lock.hcl
+# .tfstate files
+*.tfstate
+*.tfstate.*
+>>>>>>> 185-step2-Creating-API-gateway
```
Comment (mhazad) [inline faisal/.gitignore]: @faisalisafk You have committed a conflicted file here 😞

**Review by mhazad — CHANGES_REQUESTED** [review]
Pls address the feedback. This is a gross mistake.

File: faisal/.gitignore (hunk 1 above)
Comment (faisalisafk) [reply faisal/.gitignore]: Amended conflict merge on [493f04c](https://github.com/CraftsmenLtd/coe-batch-q322/pull/264/commits/493f04c75745eb9e40c36a475fccbd9f428197d5)

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**